### PR TITLE
PB-925 Add additional steps to the Master workflow to deploy to k8s after `latest` image is published on Docker Hub

### PR DESCRIPTION
**Pull request by rsaffi**

### CODE REVIEW

File: k8s/coordinator/development/config.toml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-min_sum = 1
-min_update = 3
-sum = 0.01
-update = 0.1
+min_sum_count = 1
+min_update_count = 3
+min_sum_time = 5
+min_update_time = 10
+sum = 0.5
+update = 0.9
```
Comment (little-dude) [inline k8s/coordinator/development/config.toml:13]: Were these changed on purpose? Seems unrelated to the rest of the PR

Comment (rsaffi) [reply k8s/coordinator/development/config.toml:13]: Well, I caught these changes from the default config.toml at configs/config.toml which were already merged to master.

**Comment by rsaffi** [comment]
I just tested this on a fork of our repo that I keep local here and tested against my homelab k8s cluster and it worked perfectly! :rocket:

File: k8s/coordinator/development/config.toml (hunk 1 above)
Comment (rsaffi) [reply k8s/coordinator/development/config.toml:13]: I am willing to submit these changes/updates to the `config.toml` that lives in the cluster on a separate PR, just so it doesn't get mixed with the rest of the work done on this branch.

Comment (little-dude) [reply k8s/coordinator/development/config.toml:13]: Doing it in this PR is fine for me, I was just making sure it didn't slip in by accident ;)
